Assets/MixedRealityToolkit/Utilities/MaintainBorderLightWidth.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    /// <summary>
+    /// Utility component to keep the border light width a constant size no mater the object scale.
```

Comment:
Does this require a specific shader is used? Can you add that to the comment?

Reply:
Good idea! It does require the MRTK/Standard shader, or one which follows similar naming conventions. I'll add that comment.